src/spotify.rs

```diff
@@ -137,12 +137,19 @@ impl Spotify {
             }
             Err(_) => debug!("No HTTP proxy set"),
         }
+        let ap_port = cfg.values().ap_port.unwrap_or(0);
+        if ap_port != 0 {
+            session_config.ap_port = Some(ap_port);
+        }
```

Comment:
Do we need this intermediate `0` value? Could we also do it like this?
```suggestion
        session_config.ap_port = cfg.values().ap_port;
```

Reply:
I was not sure if that would behave in the correct way (no parameter is set) when the `ap-port` option is missing in the config file. If that's the case, please feel free to apply the change, I wont be able in a couple of days.

Reply:
By default librespot sets `ap_port` to `None`: https://github.com/librespot-org/librespot/blob/4d35c5ffe222c839d1532d8afac78fe4c37043b3/core/src/config.rs#L33

However, if we wanted to be absolutely sure we could also use an `if let` expression here.